src/pages/Locations/LocationDetails/components/LocationTitle/LocationTitle.jsx

```diff
@@ -0,0 +1,44 @@
+import Gradient from "rgt";
+import TextGradientColors from "../../../../../configs/TextGradientColors";
+import {useMemo} from "react";
+import './LocationTitle.css';
+
+const LocationTitle = (props) => {
+    const randomNumber = useMemo(() => {
+        return Math.floor(Math.random() * 11);
+    }, [])
+    return (
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/mohammadnedaei/Nearby-Locations-React/pullRequest?prid=9855911)
```suggestion
  return (
```